public/locale/de.json

```diff
@@ -65,5 +75,45 @@
   },
   "EventLog": {
     "eventLog": "Ereignis Protokoll"
+  },
+  "ManagementPage": {
+    "management": "Verwaltung",
+    "structuralData": "Structural Data"
+  },
+  "ManagementGrid": {
+    "validation": {
+      "isRequired": "Feld {{field}} darf nicht leer sein.",
+      "isNotANumber": "Feld {{field}} muss eine Zahl enthalten.",
+      "isNotAnInteger": "Feld {{field}} muss eine Ganzzahl enthalten.",
+      "decimalSeperator": "Feld {{field}} muss den Dezimaltrenner '.' verwenden."
+    },
+    "csv": {
+      "unknownExperimentalFactors": "Invalides CSV. Die CSV-Datei enthält die unbekannten Versuchsfaktoren {{invalidFactors}}. Bitte legen Sie diese zuerst an.",
+      "parsingError": "Fehler beim lesen der CSV Datei: {{error}}.",
+      "noCsvFile": "Dies scheint keine CSV Datei zu sein.",
+      "emptyHeader": "Die CSV Datei enthält leere Werte in der Kopfzeile.",
+      "parsed": "{{count}} Entitäte(n) aus {{fileName}} eingelesen."
+    },
+    "encoding": "Enkodierung",
+    "fieldSeperator": "Feldtrenner",
+    "textSeperator": "Texttrenner",
+    "tabulator": "Tabulator",
+    "comma": "Komma",
+    "semicolon": "Semikolon",
+    "space": "Leerzeichen",
+    "singleQuote": "Einfache Anführungszeichen",
+    "doubleQuote": "Doppelte Anführungszeichen",
+    "csvFile": "CSV Datei",
+    "loadingSuccess": "{{count}} Entitäte(n) von {{modelName}} geladen.",
```

Comment:
Even better would it be if `Entitäte(n)` would be a placeholder and the logic would supply it with singular or plurar deoending on the count.